### PR TITLE
Frontend export sorting

### PR DESCRIPTION
**Pull request by benrudolph**
@snopoke

### CODE REVIEW

**Comment by snopoke** [comment]
:+1: once tests pass

**Comment by snopoke** [comment]
not mergable

File: corehq/couchapps/export_instances_by_domain/views/view/map.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function(doc) {
```
Comment (dimagimon) [inline corehq/couchapps/export_instances_by_domain/views/view/map.js]: Missing name in function declaration.

**Comment by snopoke** [comment]
merge once build passes

**Pull request merged**